### PR TITLE
Small README updates 🚢 

### PR DESCRIPTION
**Pull request by NicholasLin718**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Though initially inspired by COVID-19 and physical distancing, our app is design
 
 Our data is also dynamically updated to show current data as time passes, meaning TimeLine will never lose its relevance.
 
-This project was developed in 48 hours for Explore Hack 2021 by 4 graduating high school students: Wilbur Zhang, Ivan Yu, Joshua Kwong, and Nicholas Lin.
+This project was developed in 48 hours for Explore Hack 2021 by 4 graduating high school students: [Wilbur Zhang](https://github.com/arcslogger), [Ivan Yu](https://github.com/IvanYu327), [Joshua Kwong](https://github.com/JoshuaKwongTy), and [Nicholas Lin](https://github.com/NicholasLin718).
```
Comment (Arcslogger) [inline README.md]:
```suggestion
This project was developed in 48 hours for Explore Hack 2021 by 4 graduating high school students: [Wilbur Zhang](https://wilburzhang.com), [Ivan Yu](https://github.com/IvanYu327), [Joshua Kwong](https://github.com/JoshuaKwongTy), and [Nicholas Lin](https://github.com/NicholasLin718).
```

**Review by Arcslogger — CHANGES_REQUESTED** [review]

**Review by IvanYu327 — CHANGES_REQUESTED** [review]
Can u optimize

**Review by NicholasLin718 — COMMENTED** [review]
lgtm! 🚢

**Comment by Arcslogger** [comment]
analytical-engine-blueprint

**Pull request merged**